tests/parsing/ConfParserTest.py

```diff
@@ -80,7 +80,7 @@ def test_parse_nonexisting_file(self):
     def test_parse_nonexisting_section(self):
         self.assertRaises(IndexError,
                           self.uut.get_section,
-                          'inexistent section')
+                          'nonexistent section')
```

Comment:
A hyphen between non and existent, _i.e._ non-existent? [Ref](https://en.oxforddictionaries.com/definition/non-existent)